core/src/main/kotlin/org/evomaster/core/problem/rest/service/fitness/AbstractRestFitness.kt

```diff
@@ -592,7 +594,13 @@ abstract class AbstractRestFitness : HttpWsFitness<RestIndividual>() {
         val appliedLink = handleLinks(a, all,actionResults)
 
         val response = try {
-            createInvocation(a, chainState, cookies, tokens).invoke()
+            SearchTimeController.measureTimeMillis(
+                { t, res ->
+                    rcr.setResponseTime(t)
+                },
+                {createInvocation(a, chainState, cookies, tokens).invoke()}
```

Comment:
the `createInvocation()` should be not counted. rather save it to variable `call`, and then measure time of `call.invoke()`